var/spack/repos/builtin/packages/dcmtk/package.py

```diff
@@ -37,6 +37,16 @@ class Dcmtk(CMakePackage):
     variant('cxx11', default=False, description="Enable c++11 features")
     variant('stl', default=True, description="Use native STL implementation")
 
+    def patch(self):
+        # Backport 3.6.4
+        if self.spec.satisfies('%fj') and self.spec.satisfies('@:3.6.3'):
```

Comment:
```suggestion
        if self.spec.satisfies('@:3.6.3 %fj'):
```

Reply:
Thank you for your kind suggestion.
This seems better.